apps/mtabus/mta_bus.star

```diff
@@ -0,0 +1,152 @@
+"""
+BUS_ANIMATION_DURATION = 200
+
+def main(config):
+    if not config.get("key"):
+        return render.Root(
+            child = render.Text("No API key set."),
```

Comment:
It would be great if this case could result in a default graphic with mock data. Just to give users a sense of what the app will look and feel like if they move on to signing up for their own API key.

Reply:
Oooh I didn't realize! I will update, thanks.

Reply:
Smart. I'll update.

Reply:
Updated to handle cases where the user hasn't set the API key or stop ID, as well as handling the error cases. Let me know your thoughts on these various states.

### No API key set
![mta_bus_no_key_set](https://user-images.githubusercontent.com/730518/236847723-4677edd1-8466-424f-b233-e1065f9b991c.gif)

### No stop ID set
![mta_bus_no_stop_set](https://user-images.githubusercontent.com/730518/236847813-b5a02613-4a06-4c4d-b95c-4f6f4dedbbf3.gif)

### No stop results returned
![mta_bus_no_results](https://user-images.githubusercontent.com/730518/236847876-a1707124-9941-4389-b272-463ec38ecc3f.gif)

### API returns a non-200 response
![mta_bus_api_error](https://user-images.githubusercontent.com/730518/236848034-0df06178-e1a4-4af3-9470-ae21a159bdfa.gif)

Reply:
I realize you were suggesting a "default" graphic, not handling these states. To show something more-inviting to new users, do you think I should show `Next Bus ?? minutes` for the first 60 seconds, then go to the above behavior of letting them know they need to set the API key / stop ID after that?

Reply:
I think this looks great! Love the bus passing by. =)